libs/wingsdk/src/target-sim/redis.inflight.ts

```diff
@@ -76,7 +70,7 @@ export class Redis implements IRedisClient, ISimulatorResourceInstance {
     await container?.remove();
   }
 
-  public async ioredis(): Promise<any> {
+  public async rawClient(): Promise<any> {
```

Comment:
Seem to me all methods from here downwards need to be implemented in some abstract base class that all redis implementations (be they sim/tf-x/.. need to inherit from).
Perhaps just add a comment here and later when we add ElastiCache, for example, create that abstract class.